kodi_addon_checker/check_addon.py

```diff
@@ -59,6 +59,8 @@ def start(addon_path, args, all_repo_addons, config=None):
 
             check_dependencies.check_reverse_dependencies(addon_report, addon_id, args.branch, all_repo_addons)
 
+            check_files.check_print_statement(addon_report, file_index)
```

Comment:
This check should be either run just before or just after `check_py3_compatibility`.